docs/getting-started/cli.md

```diff
@@ -0,0 +1,198 @@
+---
+slug: cli
+title: CLI-only workflow (Pro)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's Pro Engine
+
+Learn how to set up Semgrep, scan your project for security issues using Semgrep's Pro Engine, and view your findings in the CLI.
+
+:::info
+For scans in your local development environment, `semgrep scan` is the preferred method to scan with Semgrep, especially for writing and testing custom rules.
```

Comment:
Semgrep provides two commands to scan with in your CLI:

- `semgrep scan` - This is the recommended command for scanning local codebases, as well as writing and testing custom rules.
- `semgrep ci` - This is the recommended command if you are scanning repositories with Semgrep as part of an organization with custom rules and policies. `semgrep ci` fetches your organization's scan configurations from Semgrep Cloud Platform.
```suggestion
For scans in your local development environment, `semgrep scan` is the preferred method to scan with Semgrep, especially for writing and testing custom rules.
```